source/PluginDev/Assets/GooglePlayGames/Platforms/Native/NativeClient.cs

```diff
@@ -288,10 +288,26 @@ private static void LaunchBridgeIntent(IntPtr bridgedIntent)
         }
         
         private AndroidJavaObject GetApiClient(GameServices services) {
-            return JavaUtils.JavaObjectFromPointer(C.InternalHooks_GetApiClient(services.AsHandle()));
+            //return JavaUtils.JavaObjectFromPointer(GooglePlayGames.Native.Cwrapper.InternalHooks.InternalHooks_GetApiClient(services.AsHandle()));
+            using (var currentActivity = GetActivity()) {
+                using (AndroidJavaClass jc_plus = new AndroidJavaClass("com.google.android.gms.plus.Plus")) {
+                    using (AndroidJavaObject jc_builder = new AndroidJavaObject("com.google.android.gms.common.api.GoogleApiClient$Builder",currentActivity)) {
+                        jc_builder.Call<AndroidJavaObject> ("addApi", jc_plus.GetStatic<AndroidJavaObject>("API"));
+                        jc_builder.Call<AndroidJavaObject> ("addScope", jc_plus.GetStatic<AndroidJavaObject>("SCOPE_PLUS_LOGIN"));
+                        AndroidJavaObject client = jc_builder.Call<AndroidJavaObject> ("build");
+                        client.Call ("connect");
+                        while(!client.Call<bool>("isConnected"))
```

Comment:
I agree with you - there is no way to do what we need to do, so this is OK for now.  can you just add to the top of the function:     
 if (!this.IsAuthenticated())                     
{
                Debug.Log("Cannot get API client - not authenticated");
                return null;
            }